src/openqaoa-core/problems/maximalindependentset.py

```diff
@@ -0,0 +1,173 @@
+import networkx as nx
+from docplex.mp.model import Model
+from typing import Union
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ..utilities import check_kwargs
+from .problem import Problem
+from .converters import FromDocplex2IsingModel
+from .qubo import QUBO
+
+
+class MIS(Problem):
+    """
+    Creates an instance of the maximal independent set (MIS) problem.
+    https://en.wikipedia.org/wiki/Maximal_independent_set
+
+    Parameters
+    ----------
+    G: networkx.Graph
+        Networkx graph of the problem 
+    penalty: float
+        Penalty for the edge constraints.
+
+    Returns
+    -------
+        An instance of the MIS problem.
+    """
+
+    __name__ = "maximal_independent_set"
+
+    def __init__(self, G, penalty:Union[int, float]=2):
+        self.G = G
```

Comment:
should this be `self._G = G`. Otherwise the property and setter method are not utilized.

Reply:
I tested the 'self._G = G' but this is the one that does not use the property and setter method. 'self.G = G' works on my local computer to use the property and setter the method. Let me know if you can reproduce it or if I'm making something wrong. @vishal-ph

Reply:
yeah, I think you are right. I will resolve the issue